src/main/java/frc/robot/DutyCycleAbsoluteEncoder.java

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot;
+
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.DutyCycle;
+import frc.robot.Constants.ConversionConstants;
+
+public class DutyCycleAbsoluteEncoder {
```

Comment:
we should only create our own encoder wrapper as a last resort, I think DutyCycleEncoder will work for us